indico/modules/events/registration/client/js/components/participant-list/ParticipantAccordion.tsx

```diff
@@ -0,0 +1,122 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {ReactNode, useState} from 'react';
+import {
+  AccordionTitle,
+  AccordionContent,
+  Accordion,
+  Icon,
+  Popup,
+  Message,
+  MessageContent,
+} from 'semantic-ui-react';
+
+import {Translate} from 'indico/react/i18n';
+
+import './ParticipantList.module.scss';
+import {ParticipantCountHidden} from './ParticipantSharedTranslations';
+import ParticipantTable from './ParticipantTable';
+import {
+  ParticipantAccordionItemProps,
+  ParticipantAccordionProps,
+  ParticipantCounterProps,
+  TableObj,
+} from './types';
+
+function ParticipantCounter({
+  totalCount,
+  hiddenCount,
+  styleName = '',
+  ...props
+}: ParticipantCounterProps) {
+  const participantCounterElement = (
+    <span>
+      {totalCount} <Icon name="user" />
+    </span>
+  );
+
+  return hiddenCount > 0 ? (
+    <div className={styleName} {...props}>
+      <Popup
+        position="left center"
+        content={<ParticipantCountHidden count={hiddenCount} />}
+        trigger={participantCounterElement}
+      />
+    </div>
+  ) : (
+    participantCounterElement
+  );
+}
+
+function ParticipantAccordionItem({table, collapsible = true}: ParticipantAccordionItemProps) {
+  const visibleParticipantsCount = table.rows.length;
+  const totalParticipantCount = table.num_participants;
+  const hiddenParticipantsCount = totalParticipantCount - visibleParticipantsCount;
+
+  const [isActive, setIsActive] = useState(!collapsible || visibleParticipantsCount > 0);
+  const handleClick = () => setIsActive(!isActive);
+
+  return (
+    <>
+      <AccordionTitle
+        active={isActive}
+        onClick={collapsible ? handleClick : undefined}
+        styleName="title"
+      >
+        {collapsible && <Icon name="dropdown" />}
+        <p>{table.title || <Translate>Participants</Translate>}</p>
```

Comment:
What about just not showing the title if we only have a single participant list? It feels a bit repetitive to have "Participants" right below "Participant List":

![image](https://github.com/user-attachments/assets/2a5a73a4-67f7-443b-a838-462e48e82eed)

Reply:
For context, the current participant list doesn't show any name for merged forms but it does show the name if there is a single unmerged regform

Reply:
I do see the point and of course I can only have the 'header' displayed when there are more than 1 tables

Reply:
Done! Original behaviour maintained. Added a 'merged' flag coming from the back-end to enable this